components/src/core/info-list-item/info-list-item.tsx

```diff
@@ -308,8 +312,8 @@ export const InfoListItem: React.FC<InfoListItemProps> = (props) => {
             {...viewProps}
         >
             <View style={[defaultStyles.statusStripe, styles.statusStripe]} />
-            {IconClass || !hidePadding ? (
-                <View style={[defaultStyles.iconWrapper, styles.iconWrapper]}>{getIcon()}</View>
+            {leftComponent || IconClass || !hidePadding ? (
+                <View style={[defaultStyles.iconWrapper, styles.iconWrapper]}>{leftComponent || getIcon()}</View>
```

Comment:
So... I see what you're saying and I can certainly add a left component that lines up with how we do it in react but I'll still need a prop to take up the space of the icon area so I can stick the avatar in there. Currently the IconClass only takes ```React.Component<{ size: number, color: string }>```. I just wanted to use this so that I didn't have to implement custom styles that mimic info-list-item for the bottomsheet header

Reply:
Made some updates here. Let me know if this is acceptable.